components/ironic/ironic-ks-user-baremetal.yaml

```diff
@@ -142,7 +142,6 @@ spec:
         - name: PATH
           value: /usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/
         - name: DEPENDENCY_SERVICE
-          value: openstack:keystone-api
```

Comment:
That is not needed as the job depends on the ironic-ks-user already (which depends on the openstack:keystone-api service).

In case of a remote keystone, this service won't exist and we can't change this via helm values in the deploy repo (as we can for the ironic-ks-user job).